src/app/config.js

```diff
@@ -0,0 +1,36 @@
+'use strict'
+
+var utils = require('./utils')
+
+var CONFIG_FILE = utils.fileKey('.browser-solidity.json')
+
+function Config (storage) {
+  this.items = {}
+
+  // load on instantiation
+  try {
+    var config = storage.get(CONFIG_FILE)
+    if (config) {
+      this.items = JSON.parse(config)
+    }
+  } catch (exception) {
```

Comment:
We should really set the editor into some fail-safe mode if this happens. Currently, we do not yet store much valuable information in the config, but as soon as we do, we should not overwrite existing config files which are malformed without the user's confirmation.